Adding homebrew installation to documentation (#6124)
flutter/website: src/docs/get-started/install/_get-sdk.md

Once you have installed any missing dependencies, run the `flutter doctor`
command again to verify that you’ve set everything up correctly.

### Downloading and installing with Homebrew

> If you have Homebrew installed on your machine, you can install Flutter using:

Reviewer: ```suggestion
If you have Homebrew installed on your machine,
you can install Flutter using the following command:
```

```terminal
$ brew install --cask flutter
```

> Then, run `flutter doctor`. That will let you know if there are
> other dependencies you need to install to use Flutter (e.g. the Android SDK).

Reviewer: ```suggestion
Then, run `flutter doctor`. That lets you know if there are
other dependencies you need to install to use Flutter, such as the Android SDK.
```
